lib/ProductOpener/Display.pm

```diff
@@ -3898,6 +3888,25 @@ HTML
 						;
 				}
 
+				if ($packager_codes{$canon_tagid}{cc} eq 'fr') {
+					$description .= <<HTML
+<p>$packager_codes{$canon_tagid}{raison_sociale_enseigne_commerciale}<br>
+$packager_codes{$canon_tagid}{adresse} $packager_codes{$canon_tagid}{code_postal} $packager_codes{$canon_tagid}{commune}<br>
+SIRET : $packager_codes{$canon_tagid}{siret} - <a href="$packager_codes{$canon_tagid}{section}">Source</a>
+</p>
+HTML
+						;
+				}
+
+				if ($packager_codes{$canon_tagid}{cc} eq 'hr') {
+					$description .= <<HTML
+<p>$packager_codes{$canon_tagid}{naziv_odobrenog_objekta_approved_establishment}<br>
+$packager_codes{$canon_tagid}{adresa_objekta_address} $packager_codes{$canon_tagid}{mjesto_i_poštanski_broj_town_postal_code} ($packager_codes{$canon_tagid}{županija_county})
```

Comment:
The accents in key names are probably causing Fatal error while critiquing "lib/ProductOpener/Display.pm": Can't call method "location" on an undefined value at /opt/perl/local/lib/perl5/PPIx/QuoteLike/Token/Interpolation.pm line 47.

Reply:
updated